test/001-version.bats

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bats
+#
+# podman-tui version test
+#
+
+load helpers
+load helpers_tui
+
+@test "podman-tui version" {
+    run_podman_tui version
+    assert "$output" =~ "podman-tui v0.2.0-dev" "expected version"
```

Comment:
This will break on every release. But as long as you realize that headache...

Reply:
I removed that simple test, I think it doesn't worth the headache.